test/validator_test.exs

```diff
@@ -5,19 +5,49 @@ defmodule ValidatorTest do
     test "checks that the user input is a non_numerical string" do
```

Comment:
Related to my previous comment about making the description more explicit: what do you think about a test description like:/
```ex
test "return an error when the user enters a non numerical string" do
```

Reply:
I like that it is succinct and explicitly says what the result of the test is, I think it is much better than the current description! The only question I have is, is the term return (or any "code" term) ok to use in test descriptions?

Reply:
That's a good questions. I think it's fine to use it if the function actually returns a something.